packages/plugins/plugin-firebase/src/methods/identify.ts

```diff
@@ -6,6 +6,13 @@ export default async (event: IdentifyEventType) => {
     await firebaseAnalytics().setUserId(event.userId!);
   }
   if (event.traits) {
```

Comment:
I think this might need to be done deeply: some of the traits are embedded objects so the "stringify" should apply to the properties of those objects too and so on 

Might be worth splitting this into its own `stringifyObject` function that gets called recursively if it finds a property in an object is another object.